### PR TITLE
Add transifex-migrate-resource script

### PR DESCRIPTION
**Pull request by laanwj**

### CODE REVIEW

File: transifex-migrate-resource.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+from requests.exceptions import RequestException
+import json
+import getpass
+import sys
+
+
+SERVER = "https://www.transifex.com"
+
+
+def get_source_entity_hash(context, key):
```
Comment (fanquake) [inline transifex-migrate-resource.py]:
```suggestion
def get_source_entity_hash(context, key) -> str:
```

File: transifex-migrate-resource.py [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+        headers=headers,
+        auth=AUTH
+    ).content
+    lang_entries = json.loads(lang_entries)
+
+    #lang_entries = [l for l in lang_entries if l['language_code']=='bg']
+
+    skip = True
+    for lang_entry in lang_entries:
+        if lang_entry['language_code']=='zh_CN.GB2312':
```
Comment (fanquake) [inline transifex-migrate-resource.py]: Could you comment why this language is special cased?

Comment (laanwj) [reply transifex-migrate-resource.py]: I don't know.

File: transifex-migrate-resource.py (hunk 1 above)
Comment (laanwj) [reply transifex-migrate-resource.py]: Let's do these kind of improvements separately.

**Comment by laanwj** [comment]
@fanquake Thanks, updated for both.

**Pull request merged**